### PR TITLE
Add FXIOS-3782 [v116] Update empty state message on remoteTabsPanel for disabled sync

### PR DESCRIPTION
**Pull request by nishant2718**
# [FXIOS-3782](https://mozilla-hub.atlassian.net/browse/FXIOS-3782) | https://github.com/mozilla-mobile/firefox-ios/issues/10049

### Description
If the user has disabled syncing tabs, we show the appropriate message in the RemoteTabsPanel empty view. This PR covers only the case where the user disables syncing. A follow up PR will adjust the message for when an account is not verified. 

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [x] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by nbhasin2** [comment]
Sorry @nishant2718 for the delay on this PR, been busy with CC work. Tagging @adudenamedruby for support 🙏🏼

File: Shared/Prefs.swift [hunk 1]
```diff
@@ -120,6 +120,7 @@ public struct PrefsKeys {
     public static let KeyInactiveTabsFirstTimeRun = "KeyInactiveTabsFirstTimeRunKey"
     public static let KeyTabDisplayOrder = "KeyTabDisplayOrderKey"
     public static let TabMigrationKey = "TabMigrationKey"
+    public static let tabSyncEnabled = "sync.engine.tabs.enabled"
```
Comment (adudenamedruby) [inline Shared/Prefs.swift:123]: We should follow the convention of all the other keys here. 
```suggestion
    public static let TabSyncEnabled = "sync.engine.tabs.enabled"
```

File: Tests/ClientTests/RemoteTabsPanelTests.swift [hunk 2]
```diff
@@ -8,6 +8,8 @@ import Common
 @testable import Client
 
 class RemoteTabsPanelTests: XCTestCase {
+    let tabsEnabledPrefKey = "sync.engine.tabs.enabled"
```
Comment (adudenamedruby) [inline Tests/ClientTests/RemoteTabsPanelTests.swift:11]: Why do you have this separate, if it's not a separate string? We should just import shared and have access to the PrefsKey that way.

**Review by adudenamedruby — APPROVED** [review]

**Comment by nishant2718** [comment]
@adudenamedruby  Thanks for the review! Updated with feedback, and will wait on BR before merging.

**Pull request merged**